src/Sphere.cs

```diff
@@ -0,0 +1,13 @@
+namespace Graphic;
+
+public class Sphere
+{
+    public Point centre { get; private set; }
+    public double radius { get; private set; }
+    
+    public Sphere(Point centre, double radius)
+    {
+        this.centre = centre;
+        this.radius = radius;
+    }
```

Comment:
```suggestion
    private Sphere(Point centre, double radius)
    {
        this.centre = centre;
        this.radius = radius;
    }
```
Also create static method-constructor